marketplaceItems/audioFocus/appResources/appData/audioFocus_app.js

```diff
@@ -0,0 +1,459 @@
+        var valid = false;
+
+        if (handController >= 0) {
+            var pose = Controller.getPoseValue(handController);
+            valid = pose.valid;
+            var controllerJointIndex;
+            if (pose.valid) {
+                if (handController === Controller.Standard.RightHand) {
+                    controllerJointIndex = MyAvatar.getJointIndex("_CAMERA_RELATIVE_CONTROLLER_RIGHTHAND");
+                } else {
+                    controllerJointIndex = MyAvatar.getJointIndex("_CAMERA_RELATIVE_CONTROLLER_LEFTHAND");
+                }
+                orientation = Quat.multiply(MyAvatar.orientation, MyAvatar.getAbsoluteJointRotationInObjectFrame(controllerJointIndex));
+                position = Vec3.sum(MyAvatar.position, Vec3.multiplyQbyV(MyAvatar.orientation, MyAvatar.getAbsoluteJointTranslationInObjectFrame(controllerJointIndex)));
+
+                // add to the real position so the grab-point is out in front of the hand, a bit
+                if (doOffset) {
+                    var offset = getGrabPointSphereOffset(handController);
+                    position = Vec3.sum(position, Vec3.multiplyQbyV(orientation, offset));
+                }
+
+            } else if (!HMD.isHandControllerAvailable()) {
+                // NOTE: keep this offset in sync with scripts/system/controllers/handControllerPointer.js:493
+                var VERTICAL_HEAD_LASER_OFFSET = 0.1 * MyAvatar.sensorToWorldScale;
+                position = Vec3.sum(Camera.position, Vec3.multiplyQbyV(Camera.orientation, { x: 0, y: VERTICAL_HEAD_LASER_OFFSET, z: 0 }));
+                orientation = Quat.multiply(Camera.orientation, Quat.angleAxis(-90, { x: 1, y: 0, z: 0 }));
+                valid = true;
+            }
+        }
+
+        return {
+            position: position,
+            translation: position,
+            orientation: orientation,
+            rotation: orientation,
+            valid: valid
+        };
+    }
+
+    // #endregion
+    // *************************************
+    // END UTILITY FUNCTIONS
+    // *************************************
+
+
+    // *************************************
+    // START INIT
+    // *************************************
+    // #region Init
+
+    var isEnabled = true;
+    var soloAvatars = {};
+
+    // #endregion
+    // *************************************
+    // STOP INIT
+    // *************************************
+
+
+    // *************************************
+    // START MAPPING FUNCTIONS
+    // *************************************
+    // #region Mapping
+
+    var lastMouseX;
+    var lastMouseY;
+    // Record the last mousePressEvent
+    function mousePressEvent(event) {
+        lastMouseX = event.x;
+        lastMouseY = event.y;
+    }
+
+
+    // Enables all the pointers created and the mapping
+    function enable() {
+        Controller.enableMapping(MAPPING_NAME);
+    }
+
+
+    // Disables all the pointers created and the mapping 
```

Comment:
you're not doing anything with the pointers anymore, right? i don't think this comment is accurate